webserver/listenstore_connection.py

```diff
@@ -0,0 +1,10 @@
+from listenstore import listenstore
+from flask import current_app
+
+def get_listenstore():
+    server = current_app.config['CASSANDRA_SERVER']
+    keyspace = current_app.config['CASSANDRA_KEYSPACE']
```

Comment:
You can move these assignments directly to dictionary.

    {
        'cassandra_server': current_app.config['CASSANDRA_SERVER'],
        'cassandra_keyspace': current_app.config['CASSANDRA_KEYSPACE'],
    }